src/DurationDatePicker.elm

```diff
@@ -289,6 +317,41 @@ isOpen (DatePicker { status }) =
             False
 
 
+{-| Returns the command to update the trigger & picker DOM elements' instances.
+Is used internally but can also be used externally in case of a changing viewport
+(e.g. onScroll or onResize).
+-}
+updatePickerPosition : Settings msg -> DatePicker -> Cmd msg
+updatePickerPosition settings (DatePicker model) =
+    case model.domLocation of
+        OutsideHierarchy { triggerDomElement, pickerDomElement } ->
+            updateDomElements triggerDomElement.id pickerDomElement.id settings (DatePicker model)
+
+        InsideHierarchy ->
+            Cmd.none
+
+
+{-| Performs the tasks of finding the trigger and picker DOM elements
+-}
+updateDomElements : String -> String -> Settings msg -> DatePicker -> Cmd msg
+updateDomElements triggerElementId pickerElementId settings (DatePicker model) =
+    Task.attempt
+        (\result ->
+            let
+                msg =
+                    case result of
+                        Ok [ triggerEl, pickerEl ] ->
+                            SetDomElements { triggerDomElement = triggerEl, pickerDomElement = pickerEl }
+
+                        _ ->
+                            -- Dom element not found
+                            NoOp
+            in
+            settings.internalMsg (update settings msg (DatePicker model))
```

Comment:
This PR needs to be updated.  We don't call the `update` directly like that anymore but just pass the Msg. So, this would probably be changed to something like:

```elm
           settings.internalMsg <|
                    case result of
                        Ok [ triggerEl, pickerEl ] ->
                            SetDomElements { triggerDomElement = triggerEl, pickerDomElement = pickerEl }

                        _ ->
                            -- Dom element not found
                            NoOp
```